Dockerfile

```diff
@@ -0,0 +1,10 @@
+# A docker file for establishing a spellcheck environment in R
+FROM rocker/tidyverse:4.2.3
```

Comment:
I would start from `rocker/r-ver:4.3.2` as that is a much smaller package. It will take a bit longer to build, but that should not be too much of an issue.